docs/book/src/developer/providers/migrations/v1.4-to-v1.5.md

```diff
@@ -16,12 +16,14 @@ maintainers of providers and consumers of our Go API.
 ## Changes by Kind
 
 ### Deprecation
+- 
+- API version`v1alpha4` is deprecated and will be removed in v1.6.
 
--
 
 ### Removals
 
--
+- API version `v1alpha3` is removed from v1.5.
```

Comment:
```suggestion
- API version `v1alpha3` is not served anymore from v1.5 (users can enable it manually in case they are lagging behind with deprecation cycles). Important: `v1alpha3` will be removed in 1.6.
```